src/store/reducers/stateCache/stateCache.types.ts

```diff
@@ -24,6 +25,7 @@ enum IncomingRequestType {
   CREDENTIAL_OFFER_RECEIVED = "credential-offer-received",
   MULTI_SIG_REQUEST_INCOMING = "multi-sig-request-incoming",
   SIGN_TRANSACTION_REQUEST = "sign-transaction-request",
+  SIGN_DATA_REQUEST = "sign-data-request",
```

Comment:
Hmm, I think we already have this with `SIGN_TRANSACTION_REQUEST` - Duke must've added it